pkg/cstor/volume/v1alpha1/cstorvolume.go

```diff
@@ -291,32 +294,35 @@ func (csr *CStorVolumeReplication) BuildConfigData() map[string]string {
 	key = fmt.Sprintf("  ConsistencyFactor")
 	value = fmt.Sprintf("  ConsistencyFactor %d", csr.ConsistencyFactor)
 	data[key] = value
-	key = fmt.Sprintf("  Replica %s", csr.ReplicaKey)
-	value = fmt.Sprintf("  Replica %s %d", csr.ReplicaKey, csr.ReplicaValue)
+	key = fmt.Sprintf("  Replica %s", csr.ReplicaID)
+	value = fmt.Sprintf("  Replica %s %d", csr.ReplicaID, csr.ReplicaGUID)
 	data[key] = value
 	return data
 }
 
 // UpdateConfig updates target configuration file by building data
-func (csr *CStorVolumeReplication) UpdateConfig() error {
+func (csr *CVReplicationDetails) UpdateConfig() error {
 	configData := csr.BuildConfigData()
 	fileOperator := util.RealFileOperator{}
 	ConfFileMutex.Lock()
 	err := fileOperator.UpdateOrAppendMultipleLines(IstgtConfPath, configData, 0644)
 	ConfFileMutex.Unlock()
+	if err == nil {
+		klog.V(4).Infof("Successfully updated istgt conf file with %v details\n", csr)
+	}
 	return err
 }
 
 // Validate verifies whether CStorReplication data read on wire is valid or not
-func (csr *CStorVolumeReplication) Validate() error {
+func (csr *CVReplicationDetails) Validate() error {
 	if csr.VolumeName == "" {
 		return errors.Errorf("volume name can not be empty")
 	}
-	if csr.ReplicaKey == "" {
+	if csr.ReplicaID == "" {
```

Comment:
let's use length to check if string is empty.

Reply:
I am slowly adapting some standard practice in go by looking into core files in standard package for the string to verify whether it is empty or the will do `if str == "" {//operation}`. If it is map or array or slice they are using `len()` so I made use of it.